client/src/components/MultiJurisdictionAudit/RoundManagement/generateSheets.test.tsx

```diff
@@ -33,64 +33,54 @@ describe('generateSheets', () => {
   beforeEach(() => mockSavePDF.mockClear())
 
   describe('downloadLabels', () => {
-    it('generates label sheets', async () => {
-      const pdf = await downloadLabels(
+    it('generates label sheets', () => {
+      const pdf = downloadLabels(
         1,
         dummyBallots.ballots,
         mockJurisdiction.name,
         mockJurisdiction.election.auditName
       )
-      const deterministicPDF = pdf
-        .replace(/CreationDate \([^)]+\)/g, '') // remove the timestamp
-        .replace(/ID \[[^\]]+\]/g, '') // remove the unique id
-      expect(deterministicPDF).toMatchSnapshot()
```

Comment:
@jonahkagan Is it no longer including the id and creation timestamp when creating the pdf?

Reply:
they aren't visible in the pdf, so we don't need to remove them anymore. they're probably still in the file metadata, but we dont test that

Reply:
Aaaah, that's awesome!! Only testing what actually matters is so awesome. :D